sw/device/lib/dif/dif_rv_timer.c

```diff
@@ -0,0 +1,194 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_rv_timer.h"
+
+#include <stddef.h>
+
+#include "rv_timer_regs.h"  // Generated.
+
+/// Converts an offset for a per-hart register at hart 0 to an offset
+/// at hart |hart|.
+static ptrdiff_t offset_by_hart(ptrdiff_t reg_offset, uint32_t hart) {
+  return reg_offset + 0x100 * hart;
```

Comment:
Could you create a define for 0x100? Or expand the function description to say that 0x100 is the size of the rv_timer CFG register.

Reply:
Done.